src/app/_components/Header/NotificationIndicator.tsx

```diff
@@ -81,124 +115,137 @@ const NotificationIndicator = ({
     setChatRoomSelect(newChatRoom);
   };
 
+  useClickAway(notificationRef, () => {
+    setOpenAlarm(false);
+  });
+
+  useEffect(() => {
+    if (!isScrollingUp) {
+      setOpenAlarm(false);
+    }
+  }, [isScrollingUp]);
+
   useEffect(() => {
-    if (newChat && newChat.receiver[userIdType] === Number(id)) {
-      setPreview(true);
+    if (
+      newNotifications ||
+      (newChat && newChat.receiver[userIdType] === Number(id))
+    ) {
+      setPreviews(newChat ? 'CHAT' : 'NOTIFICATIONS');
       startTimer();
     }
 
     return () => stopTimer();
-  }, [newChat]);
+  }, [id, newChat, newNotifications, startTimer, userIdType]);
+
+  const displatCountHandler = (count?: number) => {
+    return count
+      ? count > 99
+        ? '99+'
+        : count > 0
+        ? count.toString()
+        : ''
+      : '';
```

Comment:
삼항 연산자 대신 if문이 좀 더 직관적인 듯👀 (반영 안 해도 됌)
```suggestion
      if (count > 99) {
        return '99+';
      } else if (count > 0) {
        return count.toString();
      } else {
        return '';
      }
```